core/frontend/src/FrontendHubAccess.ts

```diff
@@ -6,70 +6,18 @@
  * @module HubAccess
  */
 
-import { AccessToken, GuidString, IModelStatus } from "@bentley/bentleyjs-core";
-import { addCsrfHeader, ChangeSet, ChangeSetQuery, IModelClient, IModelHubClient, VersionQuery } from "@bentley/imodelhub-client";
-import { IModelError, IModelVersion } from "@bentley/imodeljs-common";
-import { IModelApp } from "./IModelApp";
+import { AccessToken, GuidString } from "@bentley/bentleyjs-core";
+import { ChangesetId, IModelVersion } from "@bentley/imodeljs-common";
 
-/** @internal */
-export type ChangeSetId = string;
-
-/** @internal */
+/** @public */
 export interface IModelIdArg {

```

Comment:
@kabentley what do you think of moving this to the imodeljs-common package as well since it's used in both the backend and frontend HubAccess interfaces?  It's not too much code so not that important but may be nice for consistency.

Reply:
yes, i think that is the right thing to do

Reply:
@kabentley I didn't move this yet. I'll move it over when I handle the backend side.